benchmarks/datasets/logs/queries/pg_search/topn-agg-avg.sql

```diff
@@ -0,0 +1,2 @@
+SET paradedb.enable_aggregate_custom_scan TO on; SELECT id, message, country, severity, timestamp, paradedb.agg('{"avg": {"field": "severity"}}'::jsonb) OVER () FROM benchmark_logs WHERE message @@@ 'research' ORDER BY timestamp DESC LIMIT 10;
```

Comment:
These benchmark queries don't need the aggregate scan: can remove the GUC usage.